src/Platform/Http/Controllers/AttachmentController.php

```diff
@@ -111,6 +112,8 @@ private function createModel(UploadedFile $file, Request $request)
 
         $model->url = $model->url();
 
+        event(new UploadedFileEvent($model));
```

Comment:
Wouldn't that cause the event to be discarded twice, as it is in the File class called above?

Reply:
it's different events UploadFileEvent and Upload**ed**FileEvent